ui/component/layoutWrapperText/view.jsx

```diff
@@ -0,0 +1,98 @@
+// @flow
+import * as React from 'react';
+import { normalizeURI } from 'lbry-redux';
+import FilePrice from 'component/filePrice';
+import FileAuthor from 'component/fileAuthor';
+import FileThumbnail from 'component/fileThumbnail';
+import FileViewCount from 'component/fileViewCount';
+import FileActions from 'component/fileActions';
+import TextViewer from 'component/textViewer';
+import DateTime from 'component/dateTime';
+import RecommendedContent from 'component/recommendedContent';
+import CommentsList from 'component/commentsList';
+import CommentCreate from 'component/commentCreate';
+import MarkdownPreview from 'component/common/markdown-preview';
+import ClaimUri from 'component/claimUri';
+import FileViewerInitiator from 'component/fileViewerInitiator';
+
+type Props = {
+  uri: string,
+  metadata: StreamMetadata,
+  title: string,
+  nsfw: boolean,
+  claim: StreamClaim,
+  thumbnail: ?string,
+};
+
+function LayoutWrapperDocument(props: Props) {
+  const { uri, claim, metadata, title, nsfw, thumbnail } = props;
+  const { description } = metadata;
+
+  return (
+    <div className="">
+      <div className="main__document-wrapper">
+        <ClaimUri uri={uri} />
+
+        <div className="media__title">
```

Comment:
Is it possible for more code sharing between this and layoutwrapperfile? This block looked identical to the one above

Reply:
Yes, but I was waiting to see if they were definitely going to be there.